templates/VTEX - Template openAPI.jsonc

```diff
@@ -47,6 +47,7 @@
                         "application/json": {
                             "schema": {
                                 "type": "object",
+                                "description": "{object-description}",
```

Comment:
I don' think we should have request bodies' descriptions. Is there a specific reason you added this here?

Reply:
There was a description missing in `items` from `field2` and I ended up adding here too, I will remove this one!